swim/ping_handler.go

```diff
@@ -40,8 +42,11 @@ func handlePing(node *Node, req *ping) (*ping, error) {
 	changes, fullSync :=
 		node.disseminator.IssueAsReceiver(req.Source, req.SourceIncarnation, req.Checksum)
 
+	// Bidirectional full sync.
 	if fullSync {
-		// TODO: handle full sync
+		go func() {
```

Comment:
I don't think of this as wasteful. Full syncs are necessary because less wasteful methods: the normal dissemination protocol, has failed. I'm okay if the emergency mechanism is very wasteful, yet effective. As of now, it's half as wasteful and half as effective.

In some sense the reverse full syncs are already throttled by the average of receiving 5 pings per second. If we can't trust that this is always the case, I agree that we should throttle the reverse full syncs. One at the time seems appropriate to me.